spec/lib/percy/capybara/loaders/native_loader_spec.rb

```diff
@@ -18,50 +18,50 @@
       loader = described_class.new(page: page)
       resource_urls = loader.snapshot_resources.collect(&:resource_url)
       expect(resource_urls).to match_array([
-        "/test-css.html",
```

Comment:
Hmm, still feel the same way on this one. I like the auto-formatting, but the same problems still apply and I really don't think it's more readable for the problems it causes—even if you don't have to micromanage the front spacing, it might push the lines over line-length limits so then you'd have to try to micro-manage line-breaks.